docker/backend/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM python:3.7-bullseye
+RUN apt-get update
+
+RUN mkdir /GeoNature
+WORKDIR /GeoNature
+
+# Install apt packages
+COPY apt-packages.txt /GeoNature
+RUN xargs apt-get install -y <apt-packages.txt
+RUN wget -qO- https://deb.nodesource.com/setup_16.x | bash -
+RUN apt-get update
+RUN apt-get install -y nodejs
+# Install python packages
+RUN mkdir /GeoNature/backend
+RUN mkdir /GeoNature/frontend
+COPY backend/requirements.txt /GeoNature/backend
+COPY backend/requirements-common.txt /GeoNature/backend
+RUN pip3 install --upgrade pip
+RUN pip3 install -r backend/requirements.txt
+
+RUN echo "Europe/P" > /etc/timezone && \
+    dpkg-reconfigure -f noninteractive tzdata && \
+    sed -i -e 's/# fr_FR.UTF-8 UTF-8/fr_FR.UTF-8 UTF-8/' /etc/locale.gen && \
+    sed -i -e 's/# nb_NO.UTF-8 UTF-8/nb_NO.UTF-8 UTF-8/' /etc/locale.gen && \
+    echo 'LANG="nb_NO.UTF-8"'>/etc/default/locale && \
+    dpkg-reconfigure --frontend=noninteractive locales && \
+    update-locale LANG=nb_NO.UTF-8 \
+RUN export LC_ALL=fr_FR.UTF-8
+RUN export LANGUAGE=fr_FR.UTF-8
+RUN export LANG=fr_FR.UTF-8
```

Comment:
Arriverais-tu à fusionner ces commandes RUN ? Cela permettrait de créer moins de layers donc de générer plus rapidement l'image et de diminuer sa taille finale https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers